examples/functions-as-entry-points/functions_as_entry_points.out

```diff
@@ -0,0 +1,34 @@
+# At the command line, navigate to the directory that contains the `.bal` file and
+# and use the `ballerina run` command with the `--printreturn` option set to print
+# the return value to the standard out stream.
+
+# use the `ballerina run` command to invoke the `main` function with `add` as the
```

Comment:
Since this is the start of the sentence let's say "Use" instead of "use". Might have to do this change in all other applicable places.